allensdk/brain_observatory/ecephys/ecephys_project_cache.py

```diff
@@ -656,7 +656,8 @@ def from_warehouse(cls,
                        manifest: Optional[Union[str, Path]] = None,
                        version: Optional[str] = None,
                        cache: bool = True,
-                       fetch_tries: int = 2):
+                       fetch_tries: int = 2,
+                       timeout = 3000):
```

Comment:
Would be great to add typing info:

`timeout: int = 3000`

Reply:
Done.